binaries/go-http-daemon/main.go

```diff
@@ -62,7 +63,27 @@ func executeRequest(c echo.Context) error {
 		return parseRequestError
 	}
 
-	requestResponsePair, responseErr := request.ExecuteRequest(daemonRequest.ToRequest(), daemonRequest.Profiles, daemonRequest.Variables)
+	var req request.Request
+	if daemonRequest.RequestName != "" {
```

Comment:
This might be too nitpicky but this feels like it's own method - like `getRequest` or `getConfiguredRequest` or something like that?